mozilla/foundation.mozilla.org
# #9905 Make MozFest homepage slider cards same height

(Please ignore the false positive warnings Percy flags.)

# Description

Link to sample test page:
Related PRs/issues: #9905

### To-test locally

1. go to http://mozfest.localhost:8000/en/
2. edit MozFest homepage on Wagtail to make sure the existing three slider cards have various lengths of content
3. publish changes and QA

### Screen recording

https://user-images.githubusercontent.com/2896608/211622628-9f297c39-630e-449d-aeb8-7c0e02ca24bc.mov
